lib/Runtime/Library/JSONStack.h

```diff
@@ -15,6 +15,7 @@ namespace JSON
     class JSONStack
     {
     private:
+        Js::Var        tempValues[2];
```

Comment:
This is not a problem for JSONStack as it always cleans up the references once stringify is done with inner object/array etc.